keps/sig-node/2413-seccomp-by-default/README.md

```diff
@@ -143,14 +143,33 @@ be extended to cover the new behavior if enabled.
 
 #### Alpha
 
-- [ ] Implement the new feature gate and kubelet configuration
-- [ ] Ensure proper tests are in place
-- [ ] Update documentation to make the feature visible
+- [x] Implement the new feature gate and kubelet configuration
+- [x] Ensure proper tests are in place
+- [x] Update documentation to make the feature visible
 
 #### Alpha to Beta Graduation
 
-- [ ] Enable the feature per default
-- [ ] No major bugs reported in the previous cycle
+- [x] Enable the feature per default
+- [x] Making the feature API aware
+- [x] No major bugs reported in the previous cycle
+
+##### API Awareness
+
+We add conversion methods to the API Server with the graduation of the feature
+to beta. The beta graduation does not mean that it's enabled by default, which
+got introduced in Kubernetes v1.24.
```

Comment:
Is this accurate? [This KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/3136-beta-apis-off-by-default#non-goals) seems to say that feature gates were not part of that policy:

> Feature gates control new features (not just new APIs) and they are on by default for beta features. This KEP is not changing the lifecycle flow for feature gates.

Reply:
I think so, looking at the conversation in: https://github.com/kubernetes/kubernetes/pull/109819#discussion_r871637251